### PR TITLE
Add --no-prompt flag for update command

### PR DESCRIPTION
**Pull request by a1exsh**
Might be risky if not used carefully.

Maybe we want to extend the prompt to have a third option, to update the rest
of nodes w/o prompting, instead of just Yes/No for every node.

### CODE REVIEW

File: planb/cli.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ def extend(from_region: str,
 @click.option('--odd-host', '-O', type=str, required=True)
 @click.option('--region', type=str, required=True)
 @click.option('--force-termination', is_flag=True, default=False)
+@click.option('--no-prompt', is_flag=True)
```
Comment (slitsche) [inline planb/cli.py]: Could we specify the default explicitly?

Comment (a1exsh) [reply planb/cli.py]: Maybe, but I'm not sure we should.  I believe the semantics of flag options is `False` if not specified.  Also, we didn't do that in other places where we used `is_flag=True`, e.g. see `remote` command below.

**Comment by a1exsh** [comment]
:+1:

**Comment by slitsche** [comment]
:+1:

**Pull request merged**